doc/develop/modules.rst

```diff
@@ -569,6 +569,36 @@ Build files located in a ``MODULE_EXT_ROOT`` can be described as:
 This allows control of the build inclusion to be described externally to the
 Zephyr module.
 
+.. _modules-vulnerability-monitoring:
+
+Vulnerability monitoring
+========================
+
+The module description file :file:`zephyr/module.yml` can be used to improve vulnerability monitoring.
+
+If your module needs to track vulnerabilities using an external reference
+(e.g your module is forked from an other repository), you can use the ``vulnerabilities`` section.
+It contains the field ``external-references`` that contains a list of references that needs to
+be monitored for your module. The supported format are:
+
+- CPE (Common Platform Enumeration)
+- PURL (Package URL)
+
+.. code-block:: yaml
+
+   vulnerabilities:
+     external-references:
```

Comment:
I *think* I'd be more specific and make this two lists: `cpe-identifiers:` and `purl-identifiers:` as `external-references:` sounds really vague (unless you do want to allow people to potentially add whatever types of external-references they want, but at least now it looks like an external-identifier entry that matches neither of the two regexps gets dropped)